### DISCUSSION TITLE
Short Term consulting help with frigate and frigate card

### DISCUSSION THREAD

**Discussion by mitchell-pioneer** [blakeblackshear/frigate#10138] [Ask A Question] [answered]
**Describe what you are trying to accomplish and why in non technical terms**
I'm considering using frigate and Frigate card to build a low bandwidth video monitoring application.  It will be deployed remotely and intermittently accessed via 4G.

**Describe the solution you'd like**
Looking for some short-term consulting help to answer questions by email or Zoom on getting Frigate card configured and working.   Already have the basics of HA and frigate working, but I'm struggling with frigate card.  Also looking for best-practices and suggestions for existing projects to save.   If we select the home assistant platform, this could translate into longer-term consulting.  

**Describe alternatives you've considered**
we've used VideoLoft and other solutions that don't scale for use by 4G.

**Additional context**
Willing to pay via, credit-card, PayPal or venmo.

**Comment by NickM-27** [comment, marked as answer]
If your questions are regarding the frigate card then I'd suggest asking at https://github.com/dermotduffy/frigate-hass-card as frigate card is not maintained by this repo or frigate maintainers

**Reply by mitchell-pioneer** [reply]
Thanks Nick.   I'll take this down in a moment.  Do you know anyone who might want to help me for pay?
